tfe/data_source_workspace.go

```diff
@@ -48,6 +48,17 @@ func dataSourceTFEWorkspace() *schema.Resource {
 				Computed: true,
 			},
 
+			"file_triggers_enabled": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"trigger_prefixes": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
```

Comment:
I usually try to keep these params (more or less) organized alphabetically. Not a must, just a preference 😉

Reply:
I think I've fixed this. The other params aren't all alphabetical so I tried to guess what the intended order was and stick to it.